packages/collector/test/tracing/opentelemetry/test.js

```diff
@@ -584,176 +900,51 @@ mochaSuiteFn('opentelemetry/instrumentations', function () {
+                });
+
+                verifyExitSpan({
+                  spanName: 'otel',
+                  spans,
+                  parent: httpEntry,
+                  withError: false,
+                  pid: String(controls.getPid()),
+                  dataProperty: 'tags',
+                  extraTests: span => {
+                    expect(span.data.tags.name).to.eql('fs statSync');
                     checkTelemetryResourceAttrs(span);
                   }
                 });
               })
-            )
-          );
-      it('should trace select queries', () => sendRequestAndVerifySpans('GET', '/packages', 'SELECT * FROM packages'));
-      it('should trace batch queries', function (done) {
-        sendRequestAndVerifySpans(
-          'POST',
-          '/packages/batch',
-          "\n  INSERT INTO packages (id, name, version) VALUES (11, 'BatchPackage1', 1);\n  " +
-            "INSERT INTO packages (id, name, version) VALUES (11, 'BatchPackage2', 2);\n"
-        )
-          .then(() => {
-            done();
-          })
-          .catch(err => done(err));
-      });
-      it('should trace delete queries', () =>
-        sendRequestAndVerifySpans('DELETE', '/packages', 'DELETE FROM packages WHERE id = 11'));
+            );
+          }));
 
       it('[suppressed] should not trace', () =>
         controls
           .sendRequest({
             method: 'GET',
-            path: '/packages',
+            path: '/otel-sdk-fs',
             suppressTracing: true
           })
           .then(() => delay(DELAY_TIMEOUT_IN_MS))
           .then(() => retry(() => agentControls.getSpans().then(spans => expect(spans).to.be.empty))));
     });
 
-    describe('opentelemetry is disabled', function () {
-      globalAgent.setUpCleanUpHooks();
-      const agentControls = globalAgent.instance;
-      let controls;
-
-      before(async () => {
-        controls = new ProcessControls({
-          appPath: path.join(__dirname, './tedious-app'),
-          useGlobalAgent: true,
-          env: {
-            OTEL_ENABLED: false
-          }
-        });
-
-        await controls.startAndWaitForAgentConnection();
-      });
-
-      beforeEach(async () => {
-        await agentControls.clearReceivedTraceData();
-      });
-
-      after(async () => {
-        await controls.stop();
-      });
-      it('should not trace', () => {
-        controls
-          .sendRequest({
-            method: 'GET',
-            path: '/packages'
-          })
-          .then(() => delay(DELAY_TIMEOUT_IN_MS))
-          .then(() =>
-            retry(() => {
-              return agentControls.getSpans().then(spans => {
-                expect(spans).to.be.empty;
-              });
-            })
-          );
-      });
-    });
-  });
-
-  describe('OracleDB', function () {
-    this.timeout(1000 * 60);
-
-    describe('opentelemetry is enabled', function () {
+    describe('when Collector initialized first', function () {
```

Comment:
Could you remind me pls where the langfuse use case is tracked? We need to add this test case as well - or decide if this test is enough.
(not part of the PR)